leptonai/api/types.py

```diff
@@ -393,7 +396,8 @@ class LeptonJobSpec(BaseModel):
     parallelism: int = 1
     envs: List[EnvVar] = []
     mounts: List[Mount] = []
-    intra_job_communication: bool = False
+    ttl_seconds_after_finished: Optional[int] = None
```

Comment:
Set it to 259200 for sustain the job to 72 hrs.

Reply:
SGTM. Just checking, this does not have any side effect re stroage and computation right?

Reply:
Yes. On Web UI part this value is set to 72 hrs by default.

Reply:
done.